lib/grammar.js

```diff
@@ -335,6 +335,20 @@ var grammar = module.exports = {
       reg: /^label:(.+)/,
       format: 'label:%s'
     },
+    {
+      // RFC version 26 for SCTP over DTLS
+      // https://tools.ietf.org/html/draft-ietf-mmusic-sctp-sdp-26#section-5
+      name:'sctpPort',
+      reg: /^sctp-port:([0-9]{1,4}|[1-5][0-9]{4}|6[0-4][0-9]{3}|65[0-4][0-9]{2}|655[0-2][0-9]|6553[0-5])$/,
```

Comment:
The rfc specifies this as `sctp-port-value = 1*5<DIGIT defined in RFC4566>`, i see that this is verifying that the port is `<= 65535`, but this library probably doesn't need to handle that type of validation here. We could easily just have this regex be `/^sctp-port:(\d+)$/` and it would probably be sufficient tbh.